python/python/bystro/api/tests/test_ancestry_api.py

```diff
@@ -95,3 +98,28 @@ def test_job_data_from_beanstalkd():
 
     deserialized_values = json.decode(serialized_expected_value, type=AncestryJobData)
     assert deserialized_values == ancestry_job_data
+
+
+def test_ancestry_tsv(tmp_path):
+    pwd = os.path.dirname(os.path.abspath(__file__))
+    ancestry_file_path = Path(pwd) / "ancestry_input.json"
+    expected_results_path = Path(pwd) / "ancestry_expected_output.tsv"
+
+    expected = pd.read_csv(expected_results_path, sep="\t")
+    print("expected", expected)
+
+    # create tmp file
+    output_tsv_path = tmp_path / "output.tsv"
+    output_csv_path = tmp_path / "output.csv"
+
+    # Run the conversion for TSV
+    ancestry_json_to_format(ancestry_file_path, output_tsv_path, "tsv")
+    df1 = pd.read_csv(output_tsv_path, sep="\t")
+    print("df1", df1)
```

Comment:
Is this print and the one on 109 needed or were they for debugging? Just checking.

Reply:
Def not needed. Nice catch thanks!